tools/format_pipeline.py

```diff
@@ -0,0 +1,87 @@
+
+import sys
+import math
+import argparse
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+
+from read_nodes import include_recv_proc_nodes, include_recv_proc_edges, \
+                        read_nodes_file, read_edge_file, \
+                        read_node_features_file, read_edge_features_file
+from read_metis_partitions import read_metis_partitions
+from proc_init import run, init_process
+
+def log_params(params): 
+    print('Input Dir: ', params.input_dir)
+    print('Graph Name: ', params.graph_name)
+    print('Schema File: ', params.schema)
+    print('No. partitions: ', params.num_parts)
+    print('No. node weights: ', params.num_node_weights)
+    print('Workspace dir: ', params.workspace)
+    print('Node Attr Type: ', params.node_attr_dtype)
+    print('Edge Attr Dtype: ', params.edge_attr_dtype)
+    print('Output Dir: ', params.output)
+    print('Removed Edges File: ', params.removed_edges)
+    print('WorldSize: ', params.world_size)
+    print('Nodes File: ', params.nodes_file)
+    print('Edges File: ', params.edges_file)
+    print('Node feats: ', params.node_feats_file)
+    print('Edge feats: ', params.edge_feats_file)
+    print('Metis partitions: ', params.metis_partitions)
+
+
+def localRun(params): 
```

Comment:
this is designed for a single-machine processing. we need a distributed solution.